.github/workflows/refresh-all-metadata.yml

```diff
@@ -0,0 +1,39 @@
+name: "Refresh all metadata"
+
+on:
+  # TODO: disabled cron for now
+  # schedule:
+  #   - cron: "0 1 * * *" # Every day at 1 AM UTC
+  workflow_dispatch:
```

Comment:
Hi @mooktakim Can we add the `environment` dropdown box so we can select which environment we refresh in case we need to also run it manually for some reason..